Formula/k/kwok.rb

```diff
@@ -1,8 +1,8 @@
 class Kwok < Formula
   desc "Kubernetes WithOut Kubelet - Simulates thousands of Nodes and Clusters"
   homepage "https://kwok.sigs.k8s.io"
-  url "https://github.com/kubernetes-sigs/kwok/archive/refs/tags/v0.5.2.tar.gz"
-  sha256 "4b2dc27e4f0bb71fd70b6b829c12184f7c1e986a4692353b68f40ea142c79538"
+  url "https://github.com/kubernetes-sigs/kwok/archive/refs/tags/v0.6.0.tar.gz"
+  sha256 "4973521fc179ff2edad75c12d7862818e9cc0ae97eb85c4c160b67f9af1378ce"
```

Comment:
In line 38, may need to be modified.

``` diff
-output = OS.mac? ? shell_output(create_cluster_cmd, 1) : shell_output(create_cluster_cmd)
+output = OS.mac? ? shell_output(create_cluster_cmd) : shell_output(create_cluster_cmd)
```

Reply:
@wzshiming, thanks for the tip, simplified it a bit further in fcc3e18454b430fe93dba4b476ad7e973685fd98.